### ISSUE TITLE
mmdetection  masklrcnn to deepstream, 如何支持在deepstream获取mask

### ISSUE THREAD

**Issue by pangchao-git** [grimoire/mmdetection-to-tensorrt#118]
1 模型转化时候这样修改了Infrence.py
trt_model = mmdet2trt(cfg_path, args.checkpoint, output_names=["num_detections", "boxes", "scores", "classes","masks"], fp16_mode=args.fp16, device=args.device,enable_mask=True)
torch.save(trt_model.state_dict(), args.save_path)
2 deepstream 的配置文件

output-bbox-name=bbox
output-blob-names=num_detections;boxes;scores;classes;masks
parse-bbox-func-name=NvDsInferParseMmdet

custom-lib-path=nvdsinfer_custom_impl_fasterRCNN/libamirstan_plugin.so

**Comment by grimoire** [comment]
Not supported for now.
